docs/CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - bump to pydantic-evals,pydantic-ai 1.20.0
 - dynamic filtering of CVE data - using data deps injection with `osidb_tool`.
 - dynamic filtering of CVE data when supplied direct with static content.
-
+- implemented `suggest-statement` analysis feature
```

Comment:
`implemented` -> `enhanced`

The `suggest-statement` feature is available since `aegis-0.4.0` (and was available under a different name since the beginning).

Reply:
no probs